pom.xml

```diff
@@ -155,7 +155,7 @@
         <commons.io.version>2.4</commons.io.version>
         <commons.lang3.version>3.4</commons.lang3.version>
         <freemarker.version>2.3.28</freemarker.version>
-        <commonmark.version>0.12.1</commonmark.version>
+        <commonmark.version>0.15.1</commonmark.version>
```

Comment:
Updated because many bug fixes, specification updates and also:

Performance improvements compared to previous version 🚀:
Parsing 7-10% faster
HTML rendering 105% faster - or in other words, twice as fast!